diff -ru scan-build-2.0.13-orig/requirements.txt scan-build-2.0.13/requirements.txt
--- scan-build-2.0.13-orig/requirements.txt	2018-04-21 03:23:14.000000000 -0700
+++ scan-build-2.0.13/requirements.txt	2018-05-06 18:58:16.256660499 -0700
@@ -1 +1 @@
-typing
\ No newline at end of file
+-e .
diff -ru scan-build-2.0.13-orig/setup.py scan-build-2.0.13/setup.py
--- scan-build-2.0.13-orig/setup.py	2018-04-21 03:23:14.000000000 -0700
+++ scan-build-2.0.13/setup.py	2018-05-06 19:06:36.500808374 -0700
@@ -2,6 +2,11 @@
 # -*- coding: utf-8 -*-
 
 from setuptools import setup
+import sys
+
+install_requires = []
+if sys.version_info[0] == 2:
+    install_requires.append('typing')
 
 setup(
     name='scan-build',
@@ -14,7 +19,7 @@
     description='static code analyzer wrapper for Clang.',
     long_description=open('README.rst').read(),
     zip_safe=False,
-    install_requires=['typing'],
+    install_requires=install_requires,
     packages=['libscanbuild', 'libear'],
     package_data={'libscanbuild': ['resources/*'],
                   'libear': ['config.h.in', 'ear.c']},
