diff -Nru scan-build-2.0.15.orig/setup.py scan-build-2.0.15/setup.py
--- scan-build-2.0.15.orig/setup.py	2019-04-21 17:59:50.905813906 -0700
+++ scan-build-2.0.15/setup.py	2019-04-21 18:00:10.655860168 -0700
@@ -14,7 +14,6 @@
     description='static code analyzer wrapper for Clang.',
     long_description=open('README.rst').read(),
     zip_safe=False,
-    install_requires=['typing'],
     packages=['libscanbuild', 'libear'],
     package_data={'libscanbuild': ['resources/*'],
                   'libear': ['config.h.in', 'ear.c']},
